src/order/category/omega_complete_partial_order.lean

```diff
@@ -55,17 +55,12 @@ section
 open category_theory.limits
 
 def make_product {J : Type v} (f : J → ωCPO.{v}) : fan f :=
-@fan.mk _ _ _ _ (of (Π j, f j))
-begin
-  intro j,
-  exact continuous_hom.of_mono (pi.monotone_apply j) (λ c, rfl),
-end
+@fan.mk _ _ _ _ (of (Π j, f j)) (λ j, continuous_hom.of_mono (pi.monotone_apply j : _) (λ c, rfl))
```

Comment:
@semorrison I feel like it would be a good idea to have `P` as an explicit argument of `fan.mk`, do you have any thoughts on this?

Reply:
Yes, I agree.